src/components/ciPipeline/ciPipeline.css

```diff
@@ -480,3 +481,7 @@
 .container-image-info path[fill='#06C'] {
     fill: var(--B500);
 }
+
+.h-18 {
+    height: 18px;
+}
```

Comment:
can we move it to base css where we have all height classes?

Reply:
Done